src/pages/home/report/ReportActionItemSingle.js

```diff
@@ -88,9 +83,10 @@ const showWorkspaceDetails = (reportID) => {
 };
 
 function ReportActionItemSingle(props) {
+    const personalDetails = usePersonalDetails() || {};
     const actorAccountID = props.action.actionName === CONST.REPORT.ACTIONS.TYPE.REPORTPREVIEW && props.iouReport ? props.iouReport.managerID : props.action.actorAccountID;
-    let {displayName} = props.personalDetailsList[actorAccountID] || {};
-    const {avatar, login, pendingFields, status, fallbackIcon} = props.personalDetailsList[actorAccountID] || {};
+    let {displayName} = personalDetails[actorAccountID] || {};
```

Comment:
+1, let's not recreate this fallback object with each render

Reply:
Done ✅